acquia/cli
# CLI-864 | Unhide cs:wizard from list of ACLI commands

**Motivation**
Fixes #1192 

**Proposed changes**
Shows cs:wizard in the acli command list.


**Testing steps**

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#building-and-testing) to set up your development environment or [download a pre-built acli.phar](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#automatic-dev-builds) for this PR.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. Run `./bin/acli` and see `codestudio:wizard` in the list of available commands.
